values-zh/strings.xml

```diff
@@ -2610,7 +2607,7 @@
     <string name="dialog_filesrestart_cmdrestart">现在重启</string>
     <string name="dialog_filesrestart_cmdlater">稍后重启</string>
     <string name="settings_multiroaddecosupport">允许多重道路装饰</string>
-    <string name="loadcity_uberindicator">乌托邦</string>
+    <string name="loadcity_uberindicator">·乌托邦</string>
```

Comment:
Just to clarify, this period is intended?

Reply:
Hmmmm this is a separator, in English, for example: Sandbox · Uber Mode
Instead of the use of parentheses, because it is more suitable for Chinese grammar

Reply:
Got it.